libraries/AP_Networking/AP_Networking_port.cpp

```diff
@@ -345,6 +345,23 @@ bool AP_Networking::Port::send_receive(void)
         }
     }
 
+    if (type == NetworkPortType::UDP_SERVER && have_received) {
+        // connect the socket to the last receive address if we have one
+        uint32_t last_addr = 0;
+        uint16_t last_port = 0;
+        sock->last_recv_address(last_addr, last_port);
```

Comment:
You're ignoring the boolean return so it needs an IGNORE_RETURN(). However, perhaps this new function should return false if addr==0 and port==0 and that would clean this up a little?